CefSharp.OffScreen/ChromiumWebBrowser.cs

```diff
@@ -273,15 +285,37 @@ public class ChromiumWebBrowser : IRenderWebBrowser
         public event EventHandler NewScreenshot;
 
         /// <summary>
+        /// Top Left position of the popup.
+        /// </summary>
+        private Point popupPosition;
+
+        /// <summary>
+        ///  Size of the popup.
+        /// </summary>
+        private Size popupSize;
+
+        /// <summary>
+        /// The popup Bitmap.
+        /// </summary>
+        public Bitmap PopupImage;
```

Comment:
Size and Point are struct, which make them complicated to use as auto properties, I'm not sure making them properties is useful ? On the bitmap they are missing though, I'll fix it right now.